WORKSPACE

```diff
@@ -232,13 +214,11 @@ load("@com_github_grpc_grpc//bazel:grpc_extra_deps.bzl", "grpc_extra_deps")
 
 grpc_extra_deps()
 
-http_archive(
-    name = "com_google_private_join_and_compute",
+load("//build/com_google_private_join_and_compute:repo.bzl", "private_join_and_compute_repo")
+
+private_join_and_compute_repo(
```

Comment:
whats the advantage of doing it this way vs the former?

Reply:
keep the WORKSPACE file a little bit cleaner I guess